plugins/database/quota/src/main/java/org/apache/cloudstack/api/command/QuotaTariffCreateCmd.java

```diff
@@ -70,6 +71,7 @@ public class QuotaTariffCreateCmd extends BaseCmd {
 
     @Override
     public void execute() {
+        CallContext.current().setEventDetails(String.format("Tariff: %s, description: %s, value: %s", getName(), getDescription(), getValue()));
```

Comment:
IMO, It would be better to override `getApiResourceType()` & `getApiResourceId() ` methods. This would also link the events with the quota resources.

@harikrishna-patnala @DaanHoogland What do you think?

Reply:
Do you mean to add the fields resource_type and resource_id to the quota_tariffs table and set and return those values?
As tarriffs can change and need to keep historic data, I am not sure, but it might work. @GutoVeronezi can you chime in?

Reply:
No. I meant for Quota resource itself. Here is the javadoc for `getApiResourceId()` method.
This will require adding a key `ApiCommandResourceType.java` as well.
```java
/**
     * Commands that generate action events associated to a resource and
     * async commands that want to be tracked as part of the listXXX commands
     * need to provide implementations of the two following methods,
     * getApiResourceId() and getApiResourceType()
     *
     * getApiResourceId() should return the id of the object the async command is executing on
     * getApiResourceType() should return a type from the ApiCommandResourceType enumeration
     */
```
As per my understanding, this will add a link in UI for events view. And allows to filter events per resource type & resource id which is quota in this case.
![image](https://github.com/apache/cloudstack/assets/8760112/acb11387-52f9-47d7-9e0d-619f077fb691)

Reply:
@hsato03 can you have a look at this?

Reply:
Hi @vishesh92 @DaanHoogland,

I made the changes to allow users filter quota tariff events in UI. Could you guys take a look?